src/glfw3.jl

```diff
@@ -177,13 +177,14 @@ const JOYSTICK_LAST          = JOYSTICK_16
 # Error codes
 const NOT_INITIALIZED        = 0x00010001  # GLFW has not been initialized.
 const NO_CURRENT_CONTEXT     = 0x00010002  # No context is current for this thread.
-const INVALID_ENUM           = 0x00010003  # One of the enum parameters for the function was given an invalid enum.
-const INVALID_VALUE          = 0x00010004  # One of the parameters for the function was given an invalid value.
+const INVALID_ENUM           = 0x00010003  # One of the arguments to the function was an invalid enum value.
```

Comment:
Can you incorporate the Enum from my old error PR: https://github.com/JuliaGL/GLFW.jl/blob/2add79d0645009eb2f40ee97573cb26b222759ca/src/glfw3.jl#L178
?